### DISCUSSION TITLE
How to upload tailwind project on github pages

### DISCUSSION THREAD

**Discussion by Wellitsabhi** [tailwindlabs/tailwindcss#11865] [Help]
I made a project using html and tailwind css. & it's working on my PC. but when i upload the project to github, and try to publish it. CSS doen't work there.
I'll be thankful to all of you  for help.

**Comment by wongjn** [comment]
Consider checking that path to the CSS file is correct.

This seems like an issue with wherever you are publishing the project. If you need further assistance, consider seeking help in a discussion area more directly related to the publishing platform.
